spinalcordtoolbox/scripts/sct_dmri_display_bvecs.py

```diff
@@ -132,7 +132,7 @@ def main(argv: Sequence[str]):
         # skip b=0
         if unique_bval < BZERO_THRESH:
             continue
-        shell_colors[unique_bval] = next(colors)
+        shell_colors[unique_bval] = list(next(colors))
```

Comment:
A very picky comment would be that `tuple` is better than `list`, but that is getting to be a philosophical point.

Sorry for the regression upstream.

Reply:
Not a worry! And, thank you for taking the time to look at this downstream PR. :hearts: